baby-liminal-extension/src/frontend.rs

```diff
@@ -53,15 +53,17 @@ impl ink::env::chain_extension::FromStatusCode for BabyLiminalError {
 }
 
 /// BabyLiminal chain extension definition.
-#[ink::chain_extension]
+// IMPORTANT: this must match the extension ID in `extension_ids.rs`! However, because constants are not inlined before
+// macro processing, we can't use an identifier from another module here.
+#[ink::chain_extension(extension = 41)]
 pub trait BabyLiminalExtension {
     type ErrorCode = BabyLiminalError;
 
     /// Verify a ZK proof `proof` given the public input `input` against the verification key
     /// `identifier`.
-    // IMPORTANT: this must match the extension ID in `extension_ids.rs`! However, because constants
-    // are not inlined before macro processing, we can't use an identifier from another module here.
-    #[ink(extension = 0)]
+    // IMPORTANT: this must match the function ID in `extension_ids.rs`! However, because constants are not inlined
+    // before macro processing, we can't use an identifier from another module here.
```

Comment:
How dangerous is that we mess it up in the future in some update? 
Can we somehow reduce the danger?

Reply:
1. very dangereous
2. yup, I've already written to parity about cleaning this up